sunxspex/emission.py

```diff
@@ -18,41 +20,66 @@
 
 np.seterr(all='raise')
 
+def broken_powerlaw(x, p, q, eelow, eebrk, eehigh):
+    """
+    This routine calculates the value of the electron distribution function at electron energy x.
+    The distribution function is normalized so that the integral from eelow to eehigh is 1.
 
-# def bremsstrahlung(elecctron_dist, e_low, e_high, ):
-#     """
-#     Summary
-#
-#     Parameters
-#     ----------
-#     elecctron_dist : `astropy.units.Quantity`
-#         Description
-#     new : TYPE
-#         Description
-#
-#     Returns
-#     -------
-#     photon_flux: `astropy.units.Quantity`
-#         Photon flux in photons s^-1 keV^-1 cm^-2
-#
-#     Notes
-#     _____
-#
-#     This function solves:
-#
-#
-#     """
-#     # mc2 = 510.98d+00
-#     # clight = 2.9979d+10
-#     # au = 1.496d+13
-#     # r0 = 2.8179d-13
-#
-#     photon_flux = 0
-#
-#     return photon_flux
+    Parameters
+    ----------
+    x : np.array
+        electron energy in keV
+    p : float
+        Slope below the break (x < eebrk)
+    q : float
+        Slope above the break (x > eebrk)
+    eelow : float
+        Low cutoff
+    eebrk : float
+        Break
+    eehigh : float
+        High cutoff
+    Returns
+    -------
+    np.array
+        Power law of x
 
+    Notes
+    -----
+    A normalized, double power law electron distribution function is computed.
+    p is the lower power-law index, between eelow and eebrk, and q is the
+    upper power-law index, between eebrk and eehigh.
+
+    History
```

Comment:
GIT should track the contributions and history so I don't think we need it here.

Reply:
Hum this is showing up as a few lines up - it was regarding the History Section

Reply:
done